### PR TITLE
Update email text and job schedule for license expiration check notification.

### PR DESCRIPTION
**Pull request by catalintomai**
#### Summary
Updates text for notification email and job schedule for license expiration check notification.

#### Ticket Link
https://mattermost.atlassian.net/browse/MM-24661
Screen Shot 2020-06-08 at 4 04 28 PM

### CODE REVIEW

**Comment by catalintomai** [comment]
> If it's low effort, can you help paste a screenshot of what the email notification looks like?

sure, updated.

File: i18n/en.json [hunk 1]
```diff
@@ -2334,6 +2334,10 @@
     "id": "api.templates.post_body.button",
     "translation": "Go To Post"
   },
+  {
+    "id": "api.templates.remove_expired_license.body.message",
+    "translation": "Remove the expired license in System Console > Edition and License to stop receiving these emails."
```
Comment (jasonblais) [inline i18n/en.json:2339]:
```suggestion
    "translation": "You can remove the expired license in **System Console > Edition and License** to stop receiving these emails."
```

Comment (catalintomai) [reply i18n/en.json:2339]: We don't support markup in email templates. Trying to bold using html also doesn't work. I'll remove the background and change the text.

**Comment by srkgupta** [comment]
Removing myself from list of reviewers since this PR is obsolete. Please feel free to add me again once this becomes active again.

**Comment by catalintomai** [comment]
Cleaning up obsolete PRs.

**Pull request closed without merging**